lib/rails_upgrade_helpers/version.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module RailsUpgradeHelpers
```

Comment:
I would recommend scoping this under `Aggregate` namespace.

Since this isn't scoped under another namespace, we're editing the global namespace for `RailsUpgradeHelpers`.

This means that we could be mutating/changing expectations when `aggregate` is included in another repo since we aren't scoping our changes under `aggregate`

(i.e., if another repo defines `RailsUpgradeHelpers`, we are opening and editing that module on include of `aggregate`, which is not a good practice)